libs/shared/cloud-tasks/src/lib/cloud-tasks.ts

```diff
@@ -6,6 +6,29 @@ import { CloudTasksConfig } from './cloud-tasks.types';
 
 /** Base class for encapsulating common cloud task operations */
 export class CloudTasks {
+  /**
+   * Indicates if a queue has been configured and is enabled.
+   */
+  public get queueEnabled() {
+    // If a keyFilename was supplied, the cloud task queue can be considered enabled.
+    if (this.config.cloudTasks.credentials.keyFilename) {
+      return true;
+    }
+
+    // If we specify a local emulator is being used, then no keyFilename is required,
+    // and the task queue can be considered enabled.
+    if (this.config.cloudTasks.useLocalEmulator) {
+      return true;
+    }
+
+    // Previously, the following rule was in place, but now that the cloud task
+    // emulator has been setup, the cloud task queue can be used locally as well, so
+    // this is no longer needed.
+    //  ['stage', 'prod'].includes(this.config.env);
```

Comment:
Thank you for writing that.  But I also feel like we don't need comments on code we no longer use.  🤷

Reply:
It was mostly for awareness during review. I'll remove this. Thanks for the call out.